org.eclipse.xtext.junit4/src/org/eclipse/xtext/junit4/util/InMemoryURIHandler.java

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright (c) 2015, 2020 itemis AG (http://www.itemis.eu) and others.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.xtext.junit4.util;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.emf.ecore.resource.URIHandler;
+import org.eclipse.xtext.xbase.lib.Exceptions;
+
+/**
+ * @since 2.9
+ * 
+ * @noimplement
+ * @noreference
+ * @deprecated Use org.eclipse.xtext.testing.util.InMemoryURIHandler instead
+ */
+@Deprecated
+public class InMemoryURIHandler implements URIHandler {
```

Comment:
Out of curiosity: Is it - by chance - possible to say `class InMemoryURIHandler extends ..testing.util.InMemoryURIHandler`?

Reply:
afaik there is no relation between the plugins